<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Scientist</string>
  </property>
  <widget class="QCheckBox" name="aliveAddcheckBox">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Still alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="diedAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>100</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Died</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="diedAddSpinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2010</number>
   </property>
   <property name="value">
    <number>1900</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bornAddSpinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>71</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2010</number>
   </property>
   <property name="value">
    <number>1900</number>
   </property>
  </widget>
  <widget class="QLabel" name="bornAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>71</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Born</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="genderAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>42</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="gendrComboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>42</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="knownForLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Known for</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="knownForText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>211</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel_No">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>18</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="addToDB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>67</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitAddWindow">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>260</y>
     <width>78</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearEdit">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>260</y>
     <width>75</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>aliveAddcheckBox</zorder>
  <zorder>diedAddLabel</zorder>
  <zorder>diedAddSpinBox</zorder>
  <zorder>bornAddSpinBox</zorder>
  <zorder>bornAddLabel</zorder>
  <zorder>genderAddLabel</zorder>
  <zorder>gendrComboBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>knownForLabel</zorder>
  <zorder>knownForText</zorder>
  <zorder>pictureLabel</zorder>
  <zorder>pictureLabel_No</zorder>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>gendrComboBox</tabstop>
  <tabstop>bornAddSpinBox</tabstop>
  <tabstop>aliveAddcheckBox</tabstop>
  <tabstop>diedAddSpinBox</tabstop>
  <tabstop>knownForText</tabstop>
  <tabstop>addToDB</tabstop>
  <tabstop>exitAddWindow</tabstop>
  <tabstop>clearEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
